sdk/eventhub/azure-eventhub-checkpointstoreblob-aio/HISTORY.md

```diff
@@ -1,6 +1,7 @@
 # Release History
 
-## 1.0.0b7 (Unreleased)
+## 1.0.0 (2020-01-06)
+This release is for version bump only. No feature changes.
```

Comment:
I would probably rephrase this to something like:
```
Stable release. No new features or API changes.
```

Reply:
thanks, nice words!